### PR TITLE
Add date specification in UG

### PR DESCRIPTION
**Pull request by clarud**
Add date specification
Update help link and screenshot

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -129,33 +119,51 @@ Furthermore, certain edits can cause WardWatch to behave in unexpected ways (e.g
...
+
+**Notes on `START` and `END` Date Parameters:**  
+Dates entered on an invalid day (e.g., February 31) are automatically adjusted to the nearest valid day in the specified month.
+However, dates exceeding the maximum possible (e.g., the 32nd of any month) will result in an error.
+
+**Examples:**
+- `makeappt 1 a/Surgery s/31-02-2024-12-00 e/31-02-2024-11-00` is adjusted to `makeappt 1 a/Surgery s/29-02-2024-12-00 e/29-02-2024-11-00` (February 2024 has only 29 days).
+- `makeappt 1 a/Surgery s/32-01-2024-10-00 e/32-01-2024-11-00` will trigger an error, as no month can have a 32nd day.
+
+Invalid dates within a possible range are corrected automatically, while dates exceeding the month’s limit prompt an error.
```
Comment (niclammm) [inline docs/UserGuide.md:164]: automatically to the closest correct date

**Review by niclammm — COMMENTED** [review]
Other then that lgtm

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,79 +1,69 @@
 ---
   layout: default.md
   title: "User Guide"
-  pageNav: 3
 ---
-
-![wardwatch-logo](images/wardwatch_banner.png)
```
Comment (kyouscrap) [inline docs/UserGuide.md:7]: was it intentional to remove the entire image?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -88,13 +78,14 @@ There are 5 CLI command formats as shown below:
 1) `COMMAND_WORD`
 2) `COMMAND_WORD INDEX`
 3) `COMMAND_WORD PARAMETER`
-3) `COMMAND_WORD PARAMETERS`
-4) `COMMAND_WORD INDEX PARAMETERS`
-</box>
+4) `COMMAND_WORD PARAMETERS`
```
Comment (kyouscrap) [inline docs/UserGuide.md:81]: actually we dont have any commands that follow the COMMAND_WORD PARAMETERS format right?
should we remove point 4 entirely?

File: docs/UserGuide.md (hunk 2 above)
Comment (clarud) [reply docs/UserGuide.md:7]: Nope, good spot. I have changed it.

File: docs/UserGuide.md (hunk 3 above)
Comment (choonzies) [reply docs/UserGuide.md:85]: I think `find` and `scheduledate` command has this form!

Comment (choonzies) [reply docs/UserGuide.md:85]: I think `find` and `scheduledate` command has this form!

**Comment by shuckycheese** [comment]
lgtm

**Pull request merged**